index.html

```diff
@@ -938,168 +756,29 @@ <h3 class="bannerbox__title">Newsletter</h3>
 
             }); // end of submit for needRide
 
-            $('form[name="offerHelp"]').submit(function() {
+            $('form[name="offerHelp"]').on('submit', function() {
                 document['offerHelp'].action = "https://api.carpoolvote.com/v1.1/helper";
+            });
 
-                console.log("Submitting Form For Helper!");
-            }); // end of submit for offerHelp
-
-        });
-
-        // Hide question if not needed.
-        $('#quesDriverOrgRegNumber').hide();
-
-        // Handle Change in Different Form Attributes:
-        console.log("Setup change functions for driver.");
-        // Driver Details
-        $('#offerArea').change(function() {
-            changeOfferRideSubmit();
-        }); // Starting ZIP code for Driver Pickups.
-
-        $('#offerMiles').change(function() {
-            changeOfferRideSubmit();
-        }); // Radius for Driver
-
-        $('#driverSeats').change(function() {
-            changeOfferRideSubmit();
-        });
-
-        $('#driverWheelchair').change(function() {
-            changeOfferRideSubmit();
-        });
-
-        $('#driverFirstName').change(function() {
-            changeOfferRideSubmit();
-        });
-
-        $('#driverLastName').change(function() {
-            changeOfferRideSubmit();
-        });
-
-        $('#driverEmail').change(function() {
-            changeOfferRideSubmit();
-        });
-
-        $('#driverPhone').change(function() {
-            changeOfferRideSubmit();
-        });
-
-        $('#officialDriver').change(function() {
-            changeOfferRideSubmit();
         });
 
-        //$('#IsOfficialDriver').change(changeOfferRideSubmit());
-        // $('#offerCity').change(changeOfferRideSubmit());
-
-        // Rider
-        console.log("Setup change functions for rider");
-        // $('#rideCity').change(changeNeedRideSubmit());
-        $('#rideArea').change(function() {
-            changeNeedRideSubmit();
-        });
-        $('#rideState').change(function() {
-            changeNeedRideSubmit();
-        });
-        // $('#rideDestination').change(changeNeedRideSubmit());
-        $('#rideSeats').change(function() {
-            changeNeedRideSubmit();
-        });
-        $('#riderName').change(function() {
-            changeNeedRideSubmit();
-        });
-        $('#ridePhone').change(function() {
-            changeNeedRideSubmit();
-        });
-
-        $('.Timestamp').val(new Date());
         $('.redirect').val(window.location.origin + window.location.pathname + 'thanks.html');
```

Comment:
Why isn't this attribute just applied directly to the HTML? I don't understand why JS needs to be used for this?